### DISCUSSION TITLE
AGPL license needed for all plugins?

### DISCUSSION THREAD

**Discussion by andrewschmidt-a** [lowcoder-org/lowcoder#888] [Q&A]
I know that AGPL requires releasing the source code for all modifications to the source code. However if we develop plugins that is an extensibility scenario not modification.  do we also need to release source code?

**Comment by FalkWolsky** [comment]
No. Indeed when you develop a plugin it uses the source code of Lowcoder - but do not modify it.
The Plugin can have an other license. When you publish it - remember for example on npm - it will be "open source" anyhow, as the sources are published. Yet, if you bundle it before - it may be published as minified version.
